components/brave_wallet/browser/eip1559_transaction.cc

```diff
@@ -256,7 +256,7 @@ std::vector<uint8_t> Eip1559Transaction::GetMessageToSign(uint256_t chain_id,
   list.Append(RLPUint256ToBlob(max_priority_fee_per_gas_));
   list.Append(RLPUint256ToBlob(max_fee_per_gas_));
   list.Append(RLPUint256ToBlob(gas_limit_));
-  list.Append(to_.bytes());
+  list.Append(base::Value::BlobStorage(to_.bytes()));
```

Comment:
made EthAddress::bytes() to return const &, so some places needed a movable clone.